sources/engine/Stride.Physics/Engine/PhysicsComponent.cs

```diff
@@ -760,10 +760,8 @@ public void IgnoreCollisionWith(PhysicsComponent other, CollisionState state)
                 {
                     if(ignoreCollisionBuffer == null)
                         ignoreCollisionBuffer = new Dictionary<PhysicsComponent, CollisionState>();
```

Comment:
While we are at it, we can simplify these lines.

```csharp
ignoreCollisionBuffer ??= new()
```

Reply:
[8713632](https://github.com/stride3d/stride/pull/2299/commits/8713632f65e137ba1d048f48aa794e8f648107fb)
done

Reply:
new() can be reduced to [] ... should i do that instead?

Reply:
Yes, it's even better.